bundle.yml

```diff
@@ -35,7 +35,7 @@ projects :
 
     - loki :
         git     : https://github.com/ecmwf-ifs/loki
-        version : v0.2.1
+        version : main
```

Comment:
We should fix a version here again, to make sure that CLOUDSC itself is stable - regression testing with latest main will be triggered from Loki side

Reply:
Yes, I finally got Loki `v0.2.7` to work with this and pinned this version now. Anything before that will be able to compile the SCC-CUF transformations (enabled via `--with-cuda`), because of a Loki issue in `v0.2.6`.